docs/manuals/source/IntroductionAndTutorial/InstallingBareosWebui.rst

```diff
@@ -549,3 +548,16 @@ If you prefer to use |webui| on Nginx with php5-fpm instead of Apache, a basic w
    }
 
 This will make the |webui| accessible at http://bareos:9100/ (assuming your DNS resolve the hostname :strong:`bareos` to the NGINX server).
+
+php.ini settings
+~~~~~~~~~~~~~~~~
+
+-  The |webui| relies on date functions. Set the date.timezone directive according to the timezone your |dir| is set to.
```

Comment:
```suggestion
-  The |webui| relies on date functions. Set the date.timezone directive according to the timezone of your |dir|.
```

Reply:
I've adjusted the wording, thanks for the suggestion.